modules/admin_manual/pages/release_notes.adoc

```diff
@@ -31,6 +32,70 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.6.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.6 that need your attention.
+You can also read {owncloud-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+* To allow sharing files with very long names (> 64 characters) via federation, a migration step will run. The impact on upgrade duration depends on the number of federated shares. https://github.com/owncloud/core/pull/37835[#37835]
+
+=== PHP 7.2 Deprecation Note
+
+PHP 7.2 recently reached its https://secure.php.net/supported-versions.php[end of life] and is not maintained anymore. ownCloud Server will, therefore, drop support in one of the next minor versions as well. If you’re running on PHP < 7.3, please make sure to schedule an upgrade to PHP 7.4 as soon as possible. See the https://doc.owncloud.com/server/admin_manual/installation/system_requirements.html[system requirements] for more information.
+
+=== ownCloud Web - The New Web Frontend for ownCloud
+
+ownCloud Server 10.6 comes with the prerequisites to run the new ownCloud Web frontend as an optional component on top of it.
+
+* The new server version comes with a switcher to the new frontend ("New Design"). It will be available to users in the apps menu if the address of ownCloud Web is specified in config.php. The new frontend analogously comes with a switcher back to the classic frontend ("Classic Design") to allow users to navigate back and forth.
```

Comment:
```suggestion
* The new server version comes with a switcher to the new frontend ("New Design"). It will be available to users in the apps menu if the address of ownCloud Web is specified in config.php. Likewise, the new frontend comes with a switcher back to the classic frontend ("Classic Design") to allow users to navigate back and forth.
```